app/models/selection_process.rb

```diff
@@ -0,0 +1,7 @@
+class SelectionProcess < ApplicationRecord
+  belongs_to :invite
+  delegate :position, to: :invite
+  delegate :company, to: :position
+
+  has_many :messages, dependent: :destroy
```

Comment:
eu vi em alguns models o `dependent: :destroy`, em alguns casos até faz sentido mas em outro não, por exemplo este rsrs, não seria uma boa perder o histórico de mensagens ou outros dados importantes.

Reply:
Verdade, é que de início não tínhamos uma noção muito boa quanto qual e como as relações seriam afetadas.
Vou analisar os models e definir de forma coerente quais ações atribuir para cada um.